### PR TITLE
Some feature additions

### PR DESCRIPTION
**Pull request by purinda**
- Some commands that require `sudo` privileges will carry user environment variables using `sudo -E`
- Debian/Ubuntu system may already have a php version configured in such cases added the ability to register the new built as an alternative which can be configured via `update-alternatives`.
- Added some packages such as `libmysqlclient-dev`,etc that are required for building PHP 7 with additional configuration switches.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -2,8 +2,8 @@
 cd "$(dirname "$0")"
 
 # Dependencies
-sudo apt-get update
-sudo apt-get install -y \
+sudo -E apt-get update
+sudo -E apt-get install -y --force-yes \
```
Comment (kasparsd) [inline build.sh]: @purinda I think `-y` is already the same as `--force-yes`.

**Comment by kasparsd** [comment]
> Some commands that require sudo privileges will carry user environment variables using sudo -E

@purinda When is this required in your experience? I've never used it myself so I'm just wondering.

File: build.sh (hunk 1 above)
Comment (purinda) [inline build.sh]: @kasparsd you are right, I will remove it.

**Comment by purinda** [comment]
@kasparsd `sudo -E` is very handy when you have the user account configured with environment variables such as `http[s]_proxy`, which carries network proxy configuration to the command that gets run as `sudo`. Having `-E` is harmless even if you don't have any profile specific variables configured.

**Comment by purinda** [comment]
@kasparsd `--force-yes` is actually different to `-y` to `apt-get install`.

for example on Mint (LMDE release, which is off debian unstable branch) asks `Install these packages without verification? [y/N] y` question which is due to SSL validation as far as I understand and `--force-yes` answers yes to such question where as `-y` doesn't.. 

I can add that back to the `apt-get` command if you don't mind.

**Comment by kasparsd** [comment]
@purinda I'm thinking that the issue with environment variables is very specific to your use-case so I decide to not merge your changes in the main repo. I really appreciate your contribution, though!

Maybe you can setup an alias for `sudo` that includes the `-E` arg, see http://askubuntu.com/a/22043

**Pull request closed without merging**